misc/extension_api_validation/4.2-stable.expected

```diff
@@ -120,3 +120,13 @@ GH-87340
 Validate extension JSON: JSON file: Field was added in a way that breaks compatibility 'classes/RenderingDevice/methods/screen_get_framebuffer_format': arguments
 
 screen_get_framebuffer_format can now specify the screen it should get the format from. The argument defaults to the main window to emulate the behavior of the old function.
+
+
+GH-84792
+--------
+Validate extension JSON: Error: Field 'classes/RenderingServer/methods/environment_set_fog/arguments': size changed value in new API, from 10 to 11.
+Validate extension JSON: Error: Field 'classes/RenderingServer/methods/environment_set_fog/arguments/2': type changed value in new API, from "Color" to "enum::RenderingServer.EnvironmentFogMode".
+Validate extension JSON: Error: Field 'classes/RenderingServer/methods/environment_set_fog/arguments/3': type changed value in new API, from "float" to "Color".
+Validate extension JSON: Error: Field 'classes/RenderingServer/methods/environment_set_fog/arguments': meta was removed.
```

Comment:
I think these can be removed now since you changed the function signature to avoid shuffling the arguments around. 

If I'm correct, removing these will get rid of the CI error above.